common/script/ops/rebirth.js

```diff
@@ -5,13 +5,12 @@ import { MAX_LEVEL } from '../constants';
 import {
   NotAuthorized,
 } from '../libs/errors';
-import resetGear from '../fns/resetGear';
 import equip from './equip';
 
 const USERSTATSLIST = ['per', 'int', 'con', 'str', 'points', 'gp', 'exp', 'mp'];
 
 module.exports = function rebirth (user, tasks = [], req = {}, analytics) {
-  if (user.balance < 2 && user.stats.lvl < MAX_LEVEL) {
+  if (user.balance < 1.5 && user.stats.lvl < MAX_LEVEL) {
```

Comment:
[This commit](https://github.com/HabitRPG/habitrpg/commit/e920734648f60fde77b982aff42c1fc59e51652d) is a good example of how to [add constants](https://github.com/HabitRPG/habitrpg/commit/e920734648f60fde77b982aff42c1fc59e51652d#diff-093f1d61737be6ef20f580e7dcc66fe1R148) so that they are available under `Shared.constants.*` (like [so](https://github.com/HabitRPG/habitrpg/commit/e920734648f60fde77b982aff42c1fc59e51652d#diff-093f1d61737be6ef20f580e7dcc66fe1R148)).